### DISCUSSION TITLE
Implementing DiffusionAD

### DISCUSSION THREAD

**Discussion by lidingke** [open-edge-platform/anomalib#1013] [Feature Requests]
### What is the motivation for this task?

The projects [EfficientAD-M](https://paperswithcode.com/paper/efficientad-accurate-visual-anomaly-detection) and [DiffusionAD](https://paperswithcode.com/paper/diffusionad-denoising-diffusion-for-anomaly) have shown very promising results, but currently, there is no open-source code available online. Is there any plan to integrate these two networks?

### Describe the solution you'd like

Is there any plan to integrate these two networks?

**Comment by samet-akcay** [comment]
@lidingke, they are indeed quite promising. Due to our exiting workload, we will probably wait for the official implementations to be released.

**Reply by peacefulotter** [reply]
@samet-akcay Hi, it appears DiffusionAD has its code available at 
https://www.github.com/HuiZhang0812/DiffusionAD

An unofficial implementation of EfficientAD can be found at 
https://www.github.com/nelson1425/EfficientAD

Would love to see them in Anomalib

**Comment by samet-akcay** [comment]
Hi @peacefulotter, EfficientAD is already in anomalib, but we will see if we could implement DiffusionAD
